task_11/index.html

```diff
@@ -0,0 +1,351 @@
+	<title>Order form</title>
+	<link rel="stylesheet" type="text/css" href="css/normalize.css">
+	<link rel="stylesheet" type="text/css" href="css/style.css">
+	<link href="https://use.fontawesome.com/releases/v5.0.4/css/all.css" rel="stylesheet">
+</head>
+<body>
+	<!-- Begin header section with nav and sign in -->
+	<header class="header">
+		<button class="header-button"><i class="fas fa-long-arrow-alt-left"></i>Go back</button>
+		<img src="images/logo.png" class="logo" alt="logo">
+		<button class="header-button"><i class="fas fa-user"></i>Sign in</button>
+	</header>
+	<!-- End header section with nav and sign in -->
+	<!-- Begin main section -->
+	<section class="order">
+		<!-- Begin order summary section -->
+		<section class="order-summary">
+			<div class="order-summary-container">
+				<div class="summary-info">
+					<h3><i class="fas fa-clipboard"></i>Your order</h3>
+					<table class="summary-table">
+						<thead>
+							<tr>
+								<th>Items</th>
+								<th>Details</th>
+								<th>Price</th>
+								<th>Delivery</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td>2 documents</td>
+								<td>2 languages</td>
+								<td>696.06</td>
+								<td>48 hours</td>
+							</tr>
+						</tbody>
+					</table>
+				</div>	
+				<div class="summary-buttons">
+					<button form="main-order-form" type="submit" name="submit">Place order</button>
+					<button form="main-order-form" type="button" name="quote">Print a quote</button>
+				</div>
+			</div>
+			<div class="summary-help">
+				<h3><i class="far fa-life-ring"></i>Help center</h3>
+				<address>
+					<p>If you need assistance while placing your order, contact with our beloved Project Manager by phone or email <a href="tel:+9710555555">+971 (0) 555 555</a><a href="mailto:help@qtest.com">help@qtest.com</a></p>
+				</address>
+			</div>
+		</section>
+		<!-- End order summary section -->
+		<!-- Begin main order form section -->
+		<section class="order-form">
+			<form id="main-order-form" class="form" method="post" enctype="multipart/form-data">
+				<!-- Begin type of document subsection -->
+				<fieldset class="document-container">
+					<h1 class="h1">Select what you want to translate</h1>
+					<div class="doc-wrapper">
+			    		<label>
+			    			<input required class="visually-hidden" type="radio" name="document" value="website">
+			    			<span class="radio-wrapper">
+			    				<img src="images/website.png" alt="image for button website"><br>
+			    				Website
+			    			</span>
+			    		</label>
+			    		<label>
+			    			<input required class="visually-hidden" type="radio" name="document" value="documents">
+			    			<span class="radio-wrapper">
+			    				<img src="images/documents.png" alt="image for button documents"><br>
+			    				Documents
+			    			</span>
+			    		</label>
+			    		<label>
+			    			<input required class="visually-hidden" type="radio" name="document" value="social">
```

Comment:
Add `checked` to either of `<input type="radio"...>`. This will give a user an idea that these pictures represent a choice to make.

You may also want doing same to other two groups of radio buttons.

Reply:
Thanks!!!!)))
You're right! I had this feature on the previous version and than I've disabled it. Don't know why). 
Will change it back)
Thanks again!

Reply:
Please, mention me once you get this fixed. I shall then close the case with approval.
Thanks!

Reply:
Fixed!
Thanks a lot for suggestions and for your help) I appreciate it so much!

Reply:
LGTM, approving!
Please, update `gh-pages`